File: app/src/androidTest/java/com/hulkdx/moneymanagerv2/injections/TestComponent.java
package com.hulkdx.moneymanagerv2.injections;

import com.hulkdx.moneymanagerv2.injection.component.ApplicationComponent;
import javax.inject.Singleton;
import dagger.Component;

@Singleton
@Component(modules = ApplicationTestModule.class)
public interface TestComponent extends ApplicationComponent {

}

File: presentation/src/androidTest/java/com/hulkdx/moneymanagerv2/di/TestComponent.kt
package com.hulkdx.moneymanagerv2.di

import com.hulkdx.moneymanagerv2.di.components.ApplicationComponent
import dagger.Component
import javax.inject.Singleton

/**
 * Created by Mohammad Jafarzadeh Rezvan on 06/03/2019.
 */
@Singleton
@Component(modules = [TestModule::class])
interface TestComponent: ApplicationComponent {

}